eng/SourceBuild.props

```diff
@@ -5,19 +5,4 @@
     <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
   </PropertyGroup>
 
-  <Target Name="ApplySourceBuildPatchFiles"
```

Comment:
Keeping this part for future patches could be helpful, since it would do nothing if there is not patches present.

Reply:
We don't intend to have/allow patches going forward.  We want the product to support source-build natively.  Adding source build legs to the CI will help ensure the product is source-buildable when changes are introduced.